Makefile

```diff
@@ -89,7 +89,7 @@ help:
 	printf "\n"
 
 ## Installs a development environment using docker-compose
-install: generate-passwords create-certs compose
+install: create-certs compose generate-passwords
 
 ## Installs a development environment using docker-compose and pulls security tests' images
 install-pull-images: generate-passwords create-certs compose pull-images
```

Comment:
I think we might need to change the order here as well.

Reply:
Good catch! I have just added it here: b45390c